spec/spec_helper.rb

```diff
@@ -39,11 +39,11 @@ def committed_at
     require_relative '../test/test_helpers/configuration'
     require_relative '../test/test_helpers/storage'
 
-    TestHelpers::Storage::Mock.mock_storage_client!
+    TestHelpers::Storage::Mock.mock_storage_clients
```

Comment:
Not a fan of letting the detail that there are now two storage clients leak into the naming. Also, this is meant to use just one storage per invocation, isn't it?